texar/utils/utils.py

```diff
@@ -605,6 +608,37 @@ def get_instance_kwargs(kwargs: Kwargs, hparams: ParamDict) -> Kwargs:
     return kwargs_
 
 
+def get_output_size(instance_: Optional[Module]) -> Optional[int]:
+    r"""Return the final dimension of output size from :attr:`instance_`.
+
+    If type of :attr:`instance_` is among the common types, the final
+    dimension of output size will be computed.
+
+    Args:
+        instance_: A :class:`~torch.nn.Module` instance from
+            which to compute output size.
+
+    :returns:
+        int (optional): The final dimension of the output size.
```

Comment:
And here, what's in front of the colon (`:`) is exactly the type description, which is the legitimate use of colons in a `Returns:` directive.

Reply:
This is not resolved. The `:returns:` here could be changed back to `Returns:` without problem. Could you check to see if this is the case?

Reply:
Oh sorry, I thought the content containing the colon (:) needed to labeled `:returns:`, I'll fix this, thanks!